packages/block-editor/src/store/selectors.js

```diff
@@ -2034,11 +2034,8 @@ export const getInserterItems = createSelector(
 			? getReusableBlocks( state )
 					.filter(
 						( reusableBlock ) =>
-							// Filter to either fully synced patterns (sync_status === 'fully'),
-							// or old school reusable blocks (sync_status === '').
-							reusableBlock.meta?.sync_status === 'fully' ||
-							reusableBlock.meta?.sync_status === '' ||
-							! reusableBlock.meta?.sync_status
+							reusableBlock.wp_pattern_sync_status === '' ||
+							! reusableBlock.wp_pattern_sync_status
```

Comment:
Didn't we add the explicit check for `fully` because a bug where it could be set landed in 16.1?

Reply:
ah, yeh, have put back and clarified the comment.

Reply:
I think we've now lost the rename to `wp_pattern_sync_status` that the original change made here.

Reply:
my local commit fix for this didn't push for some reason - sorted now.